packages/windows-terminal.vm/tools/chocolateyuninstall.ps1

```diff
@@ -1,9 +1,11 @@
 $ErrorActionPreference = 'Continue'
 Import-Module vm.common -Force -DisableNameChecking
 
-$toolName = 'Microsoft Windows Terminal'
+$toolName = 'Windows Terminal'
 $category = 'Productivity Tools'
 $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
 $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
 
-Remove-Item $shortcut -Force -ea 0 | Out-Null
\ No newline at end of file
+Remove-Item $shortcut -Force -ea 0 | Out-Null
+Remove-Item $toolDir
```

Comment:
why not using the helper here?
```
VM-Uninstall $toolName $category
```

If there is a reason we can't use the helper function, at least we should remove the `$toolDir` with:
```
Remove-Item $toolDir -Recurse -Force -ea 0 | Out-Null
```